caasp-istio-proxyv2-image/caasp-istio-proxyv2-image.kiwi

```diff
@@ -16,7 +16,7 @@
         name="caasp/v5/istio-proxyv2"
         tag="%%PKG_VERSION%%"
         maintainer="SUSE Containers Team &lt;containers@suse.com&gt;">
-        <entrypoint execute="/usr/bin/pilot-agent"/>
+	<entrypoint execute="/usr/local/bin/pilot-agent"/>
```

Comment:
nit: looks like there is a little indentation issue here

Reply:
thanks!